Build failure: missing package 'transformers'

When trying to build the project with...

```
$ stack build
```

... I get the following error:

```
/home/user/devel/disclaimer/Main.hs:16:1: error:
    Failed to load interface for `Control.Monad.Trans.Writer'
    It is a member of the hidden package `transformers-0.5.2.0'.
    Perhaps you need to add `transformers' to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
```

Needs to change dependency on `mtl` to `transformers` in [package.yaml](https://github.com/haskell-dump/disclaimer/blob/master/package.yaml).

--- a/Main.hs
+++ b/Main.hs
@@ -1,18 +1,4 @@
 import Control.Applicative
-
--- import Control.Monad.Writer
-	{--
-	Prelude> :l Main.hs
-
-	Main.hs:2:8:
-		Could not find module ‘Control.Monad.Writer’
-		Perhaps you meant
-		  Control.Monad.Fix (from base-4.8.2.0)
-		  Control.Monad.Zip (from base-4.8.2.0)
-		Use -v to see a list of the files searched for.
-	Failed, modules loaded: none.
-	Prelude>
-	--}
 import Control.Monad.Trans.Writer
 import Data.Function
 

--- a/package.yaml
+++ b/package.yaml
@@ -6,4 +6,4 @@ executables:
     main: Main.hs
     dependencies:
       - base
-      - mtl
+      - transformers

